### PR TITLE
Changelog: v3.0.0

### PR DESCRIPTION
**Pull request by manics**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,193 @@
...
+- Bump eslint-plugin-jest from 25.2.4 to 25.3.0 [#147](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/147) ([@dependabot](https://github.com/dependabot))
+- Bump eslint from 8.2.0 to 8.3.0 [#146](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/146) ([@dependabot](https://github.com/dependabot))
+- Bump nock from 13.2.0 to 13.2.1 [#144](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/144) ([@dependabot](https://github.com/dependabot))
+- Bump eslint from 8.1.0 to 8.2.0 [#143](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/143) ([@dependabot](https://github.com/dependabot))
+- Bump nock from 13.1.4 to 13.2.0 [#142](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/142) ([@dependabot](https://github.com/dependabot))
+- Bump eslint-plugin-jest from 25.2.2 to 25.2.4 [#141](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/141) ([@dependabot](https://github.com/dependabot))
+- Bump eslint from 7.32.0 to 8.1.0 [#138](https://github.com/jupyterhub/action-major-minor-tag-calculator/pull/138) ([@dependabot](https://github.com/dependabot))
```
Comment (manics) [inline CHANGELOG.md:183]: Is it useful to have all these dependabot PRs in the changelog?

Comment (consideRatio) [reply CHANGELOG.md:183]: I don't think dev dependency bumps is worth being mentioned, and if there are too many of the same prod bumps, its too noisy as well.

:+1: for at least removing all dev dependency stuff as a start and evaluating if there is further cleanup to do

Comment (consideRatio) [reply CHANGELOG.md:183]: I've omitted irrelevant github action bumps as well from changelogs, but probably not consistently.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,193 @@
...
+
+### v3.0.0 - 2023-11-20
+
+([full changelog](https://github.com/jupyterhub/action-major-minor-tag-calculator/compare/v2.0.0...v3.0.0))
+
+#### Breaking changes
+
+This action now uses Node.js 20.
+
+## Maintenance and upkeep improvements
```
Comment (consideRatio) [inline CHANGELOG.md:11]: `github-activity --heading-level=3` makes the output heading levels be correct. This and some other heading levels below needs `##` added.

Comment (manics) [reply CHANGELOG.md]: Done, and I've removed the dependabot updates and linked out to them instead https://github.com/jupyterhub/action-major-minor-tag-calculator/pulls?q=is%3Apr+merged%3A2021-10-25..2023-11-20+author%3Aapp%2Fdependabot

**Review by consideRatio — APPROVED** [review]

**Comment by consideRatio** [comment]
Thank you @manics for getting this done!!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,32 @@
+## v3
+
+### v3.0.0 - 2023-11-21
```
Comment (manics) [inline CHANGELOG.md:3]:
```suggestion
### v3.0.0 - 2023-11-23
```

**Comment by consideRatio** [comment]
For reference, these are the bump PRs in the jupyterhub org:
- https://github.com/jupyterhub/docker-image-cleaner/pull/145
- https://github.com/jupyterhub/jupyterhub/pull/4639
- https://github.com/jupyterhub/configurable-http-proxy/pull/513

**Pull request merged**